docker/kyuubi-pod.yaml

```diff
@@ -17,20 +17,25 @@
 
 # Usage:
 # Start:
-#   kubectl apply -f https://raw.githubusercontent.com/apache/incubator-kyuubi/master/docker/example.yaml
-# Connect:
+#   kubectl apply -f https://raw.githubusercontent.com/apache/incubator-kyuubi/master/docker/kyuubi-pod.yaml
```

Comment:
why is the name changed?

Reply:
This `yaml` is used to create pod, example contains other kinds of `yaml` like `service`.